Source/Core/Core/HW/SI/SI_DeviceGCSteeringWheel.cpp

```diff
@@ -81,26 +81,33 @@ void CSIDevice_GCSteeringWheel::SendCommand(u32 command, u8 poll)
 
   if (wheel_command.command == CMD_FORCE)
   {
-    // 0 = left strong, 127 = left weak, 128 = right weak, 255 = right strong
-    unsigned int strength = wheel_command.parameter1;
-
-    // 06 = motor on, 04 = motor off
-    unsigned int type = wheel_command.parameter2;
-
     // get the correct pad number that should rumble locally when using netplay
     const int pad_num = NetPlay_InGamePadToLocalPad(m_device_number);
 
     if (pad_num < 4)
     {
-      if (type == 0x06)
+      // Lowest bit is the high bit of the strength field.
+      const auto type = ForceCommandType(wheel_command.parameter2 >> 1);
```

Comment:
I don't suppose you know of any other values that can be in that place?
YAGCD unfortunately has nothing on the steering wheel yet, so I cannot really verify whether this (or the 9-bit value for strength) are actually correct or have something else going on...

Reply:
https://github.com/ExtremsCorner/libogc-rice/blob/master/libogc/si_steering.c#L289 implies it's 9 bits. He said it was derived from decompiled code.

Reply:
Oh, right, totally forgot about libogc as potential reference. LGTM then 👍